sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -63,14 +63,26 @@
     c.client_net_address as client_address,
     sess.host_name as host_name,
     sess.program_name as program_name,
+    sess.is_user_process as is_user_process,
+    CONVERT(
+        NVARCHAR, TODATETIMEOFFSET(at.transaction_begin_time, DATEPART(TZOFFSET, SYSDATETIMEOFFSET())), 126
+    ) as transaction_start,
+    at.transaction_state as transaction_state,
+    at.transaction_type as transaction_type,
     {exec_request_columns}
 FROM sys.dm_exec_sessions sess
     INNER JOIN sys.dm_exec_connections c
         ON sess.session_id = c.session_id
     INNER JOIN sys.dm_exec_requests req
         ON c.connection_id = req.connection_id
+    LEFT JOIN sys.dm_tran_session_transactions st
+        ON sess.session_id = st.session_id
+    LEFT JOIN sys.dm_tran_active_transactions at
+        ON st.transaction_id = at.transaction_id
```

Comment:
How does adding transaction relation relate to the motivations of this PR?

> Customers are seeing background processes in some places in the UI, and these have a disproportionally high duration compared to other samples.

We have an ongoing discussion about how to monitor locks and open transactions. Shall we wait with adding the join until this discussion is resolved?

Reply:
> How does adding transaction relation relate to the motivations of this PR? 

The motivation is to improve our activity collection to add 2 pieces of information, whether a spid is a user or background process, and whether it's involved in a transaction (information we have in our postgres integration). 

> Shall we wait with adding the join until this discussion is resolved?

If we wait, we are delaying getting this in the hand of customers by 6+ weeks. I'm not sure what's controversial about adding transaction information to our activity events?